prowler/providers/aws/services/organizations/organizations_delegated_administrators/organizations_delegated_administrators.metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+  "Provider": "aws",
+  "CheckID": "organizations_delegated_administrators",
+  "CheckTitle": "Check if trusted delegated administratos",
```

Comment:
Could you include a more descriptive title like "Check if the AWS Organization for this account has trusted delegated administrators". What do you think?